server/routes/overview/mappers/supportNeedsMapper.test.ts

```diff
@@ -0,0 +1,51 @@
+import type { SupportNeeds } from 'viewModels'
+import { toSupportNeeds } from './supportNeedsMapper'
+
+describe('supportNeedsMapper', () => {
+  it('should map to SupportNeeds', () => {
+    // Given
+    const learnerProfile = {
+      prn: 'G6123VU',
+      establishmentId: 'MDI',
+      lddHealthProblem: 'Learner considers himself or herself to have a learning difficulty.',
+      languageStatus: 'Bilingual',
+      primaryLDDAndHealthProblem: 'Visual impairment',
+      additionalLDDAndHealthProblems: [
+        'Hearing impairment',
+        'Social and emotional difficulties',
+        'Mental health difficulty',
+      ],
+    }
+    const learnerNeurodivergence = {
+      prn: 'G6123VU',
+      establishmentId: 'MDI',
+      neurodivergenceSelfDeclared: ['Dyslexia'],
+      selfDeclaredDate: '2022-02-18',
+      neurodivergenceAssessed: ['No Identified Neurodiversity Need'],
+      assessmentDate: '2022-05-18',
+      neurodivergenceSupport: ['Writing support'],
+      supportDate: '2022-02-18',
+    }
+
+    const expectedSupportNeeds = {
+      languageSupportNeeded: false,
```

Comment:
I've no idea how to map this yet.

Reply:
I'm doing a straight mapping from `languageStatus` for now. The business are going to clarify what the requirements are here.